qiskit_ibm_runtime/api/session.py

```diff
@@ -51,20 +54,35 @@
 
 def _get_client_header() -> str:
     """Return the client version."""
-    try:
-        client_header = "qiskit/" + pkg_resources.get_distribution("qiskit").version
-        return client_header
-    except Exception:  # pylint: disable=broad-except
-        pass
-
-    qiskit_pkgs = ["qiskit-terra", "qiskit-aer", "qiskit-ignis", "qiskit-aqua"]
-    pkg_versions = {"qiskit-ibm-runtime": ibm_runtime_version}
+
+    qiskit_pkgs = [
+        "qiskit-terra",
+        "qiskit-aer",
+        "qiskit-ignis",  # TODO: remove this?
+        "qiskit-aqua",  # TODO: remove this?
```

Comment:
Yeah I think these can be removed since both of those repos have been deprecated for a while

Reply:
Done!